minecraft_servers/muffintime/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "server_name": "muffintime",
+  "nice_name": "The MuffinTime Minecraft",
+  "direct_ip": "mc.muffintime.tk",
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "web": "https://mc.muffintime.tk"
```

Comment:
Not really a website, if you have a normal website, link it; if you don't, remove please

Reply:
I removed it.